### PR TITLE
[mk] New Language, removing excess dot from sr_Latn, sr_Cyrl and lt

### PR DESCRIPTION
**Pull request by keljtanoski**
This pull request adds [mk] language translations

### CODE REVIEW

File: referents.md [hunk 1]
```diff
@@ -39,7 +39,7 @@
 | ko | |
 | lt | |
 | lv | |
-| mk | |
+| mk | [@keljtanoski](https://github.com/keljtanoski) |
```
Comment (caouecs) [inline referents.md:42]: Thank you for proposing as a referent, but we like to validate before

**Review by andrey-helldar — APPROVED** [review]

File: src/mk/auth.php [hunk 2]
```diff
@@ -12,6 +12,6 @@
     |
     */
 
-    'failed'   => 'These credentials do not match our records.',
-    'throttle' => 'Too many login attempts. Please try again in :seconds seconds.',
+    'failed'   => 'Овие акредитиви не се совпаѓаат со нашите записи.',
+    'throttle' => 'Премногу обиди за најавување. Обидете се повторно за :seconds секунди.',
```
Comment (caouecs) [inline src/mk/auth.php:16]: Can you add : 'password' => 'The provided password is incorrect.', ?

File: referents.md (hunk 1 above)
Comment (andrey-helldar) [inline referents.md:42]: @caouecs, we have a replenishment 🙂

**Review by caouecs — CHANGES_REQUESTED** [review]
Hello,
thank you

File: referents.md (hunk 1 above)
Comment (keljtanoski) [reply referents.md:42]: @caouecs, Gotcha, sorry about this :)

**Comment by caouecs** [comment]
Thank you

**Comment by caouecs** [comment]
@keljtanoski If you are always ok to be referent on mk language, it's ok for us

**Comment by keljtanoski** [comment]
@caouecs 
Sure, i am ok to be referent on mk language.

Thank you.

**Pull request merged**